OWML.ModHelper.Menus/ModConfigMenu.cs

```diff
@@ -174,6 +181,12 @@ private void AddTextInput(string key, int index)
             textInput.Show();
         }
 
+        private void AddInputInput(string key, int index)
+        {
+            IModInputInput inputInput = AddInputInput(_inputInputTemplate.Copy(key), index);
```

Comment:
Could be changed to "var"

Reply:
I kinda prefer explicit types. Are there any conventions that say to prefer var in certain cases?

Reply:
Not sure about conventions, but the rest of the codebase already follows this. No need to specify the type if it can be inferred.

Reply:
I follow the conventions recommended by ReSharper. Here's their article about implicit types (var): https://www.jetbrains.com/help/resharper/Using_var_Keyword_in_Declarations.html
Btw, I recommend ReSharper, it's a plugin for VS which suggests code improvements. It's free for a month: https://www.jetbrains.com/resharper/